css/terminal7.css

```diff
@@ -767,11 +749,20 @@ audio { display: none; }
     text-align: left;
     padding-left: 1.5em;
     text-shadow: none;
+    z-index: 20;
+    border-bottom: 1px rgba(255, 255, 255, 0.7) solid;
+    border-right: 0;
+    border-top: 0;
+    -webkit-border-top-right-radius: 6px;
```

Comment:
This and the next line are not needed anymore. It's an ancient hack and all modern browsers have support for `border-top-right-radius`. In the future, use https://caniuse.com to check if you need any browser specific rules

Reply:
It's not clear, what lines we're talking about
- 4th line will be changed to border-top-right-radius
- lines 2-3 can be removed